monkey/tests/unit_tests/infection_monkey/network_scanning/test_ping.py

```diff
@@ -87,7 +87,7 @@ def set_os_windows(monkeypatch):
 @pytest.mark.usefixtures("set_os_linux")
 def test_linux_ping_success(patch_subprocess_running_ping_with_ping_output):
     patch_subprocess_running_ping_with_ping_output(LINUX_SUCCESS_OUTPUT)
-    result = ping("192.168.1.1", 1.0)
+    result = _ping("192.168.1.1", 1.0)
```

Comment:
Unit tests should avoid calling internal functions and exercise the public interface instead.